cmd/gossamer/main_test.go

```diff
@@ -193,7 +227,56 @@ func TestInvalidCommand(t *testing.T) {
 	}
 }
 
-// TODO: TestGossmaerCommand test "gossamer" does not error
+func TestGossamerCommand(t *testing.T) {
+	basePort := 7000
+	currentDir, err := os.Getwd()
+	require.Nil(t, err)
+
+	genesisPath := filepath.Join(currentDir, "../..", "node/gssmr/genesis.json")
+
+	tempDir, err := ioutil.TempDir("", "gossamer-stress-")
+	require.Nil(t, err)
+
+	gossamer := runTestGossamer(t,
+		"init",
+		"--datadir", tempDir,
+		"--genesis", genesisPath,
+		"--force",
+	)
+
+	stdout, stderr := gossamer.GetOutput()
+	log.Println("init gossamer output, ", "stdout", string(stdout), "stderr", string(stderr))
```

Comment:
is there a reason `log` is used instead of `t.Log`? not really a big deal, just wondering

Reply:
Hmmmm, that was because I was not seeing the logs right and tried to change it in order to see a different state. Should be updated.